docs/hosted-orchestration/building-images.md

```diff
@@ -0,0 +1,168 @@
+
+```
+...
+Successfully built e32703162dd4
+```
+
+Read more about docker build command.
+
+Congratulations, you’ve just built your first image! Now we need to store it somewhere to make it available for Sauce Labs.
+
+## Pushing to a Registry
+
+In order to allow Sauce Labs to use your custom image, store it in a public Docker Registry. The easiest mechanism is to create an account on Docker Hub because Docker Hub allows you to store unlimited public images for free. If your organization is already using Docker Hub you can use your existing account.
+
+Note: To use an image with the Sauce Labs Hosted Orchestration you must have a public repository. If you want to keep your image private, refer to the Using Docker Authenticated Pulls document for instructions.
+
+The example uses Docker Hub, but it is possible to use different registries if you prefer. Adapt the example based on the registry you are using.
+
+### Preparing the image for the registry
+
+Log in to Docker Hub with your account and create a new repository on the add repository page. It is best practice to use a pattern similar to ```<project-name>-<container-name>``` for a repository name (for example, ```sl-demo-docker-primary```).
+
+Next, rebuild your image using your account and repository name:
+
+```
+$ docker build -t saucelabs/sl-demo-docker-primary:0.0.1 <path-to-dockerfile>
+```
+
+The -t key specifies the name and tag of the new image:
+
+- saucelabs - our account in Docker Hub
+- sl-demo-docker-primary - repository name
+- 0.0.1 - tag (version) of the image. Always update the tag if you change something in a Dockerfile otherwise you might have unpredictable results.
+
+### Pushing the image to the registry
+
+Push the image to Docker Hub:
+
+```
+$ docker login
+$ docker push saucelabs/sl-demo-docker-primary:0.0.1
+```
+
+Note: First, we use docker login to authenticate in Docker Hub. If you use a registry other than Docker Hub, refer to the related documentation about how to push images to that registry.
+
+### Using your image in Sauce Labs 
+After the image is successfully pushed it is available for use in Hosted Orchestration. Create a SauceCTL configuration like the one below. For more information on running tests see [Running Tests](./running-tets)
+
+<Tabs
+     defaultValue="SauceCTL"
+     values={[
+       {label: 'SauceCTL', value: 'SauceCTL'},
+     ]}>
+  <TabItem value="SauceCTL">
+
+  ```yaml
+  apiVersion: v1alpha
+  kind: htexec
+  sauce:
+  region: us-west-1
+  suites:
+    - name: run sauce test
+      image: saucelabs/sl-demo-docker-primary:0.0.1
+      entrypoint: "mvn test"
+      files:
+        - src: "runsauce.json"
+          dst: "/workdir/runsauce.json"
+      artifacts:
+        - "/path/inside/container/file.log"
+      env: 
+        KEY: value
+   ```
+
+  Then run with
+  ```bash
+    saucectl run
```

Comment:
Just fyi that this only works if the config name follows a specific pattern and resides in a subfolder like so: `.sauce/config.yml`

Otherwise, the user has to point to the config file explicitly: `saucectl run -c myconfig.yml`